.vscode/extensions.json

```diff
@@ -1,6 +1,7 @@
 {
   "recommendations": [
     "rvest.vs-code-prettier-eslint",
-    "streetsidesoftware.code-spell-checker"
+    "streetsidesoftware.code-spell-checker",
+    "DigitalBrainstem.javascript-ejs-support"
```

Comment:
Without this plugin VSCode will not highlight the EJS parts of our HTML template properly (they'll just be HTML, which is fine -- but its nice when the lines are highlighted properly).

with plugin:

<img width="363" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/187813/78e84097-11aa-42ab-93b5-91c4d9dd1d55">

without: 
<img width="357" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/187813/718a26a3-930e-4168-b50f-fc5835077ae3">